toolkits/landcover/impl/landsat8.js

```diff
@@ -59,6 +77,7 @@ var Landsat8 = function() {
 Landsat8.prototype = Object.create(Dataset.prototype);
 
 Landsat8.prototype.COMMON_BAND_NAMES = COMMON_BAND_NAMES;
+Landsat8.prototype.TASSELED_CAP_COEFFICENTS = TASSELED_CAP_COEFFICENTS;
```

Comment:
I think a more idiomatic way of doing this would be to have a function `getTasseledCapCoefficients()`) that gets overridden instead copying a reference to the a constant into each instance of Landsat8.

Reply:
Done